perf/src/sigverify.rs

```diff
@@ -800,17 +829,17 @@ mod tests {
         tx.signatures = vec![sig; NUM_SIG];
         tx.message.account_keys = vec![];
         tx.message.header.num_required_signatures = NUM_SIG as u8;
-        let mut packet = Packet::from_data(None, tx).unwrap();
+        let mut packet = BytesPacket::from_data(None, tx).unwrap();
```

Comment:
why change these and the other tests to BytesPacket?

Reply:
How do we know that sigverify works on `BytesPacket` otherwise? This module is mostly used by TPU (turbine has its own sigverify 🫠 ).

Reply:
fair!